linera-rpc/src/grpc/server.rs

```diff
@@ -510,7 +509,7 @@ where
     #[instrument(target = "grpc_server", skip_all, err, fields(nickname = self.state.nickname(), chain_id = ?request.get_ref().chain_id()))]
     async fn handle_certificate(
         &self,
-        request: Request<Certificate>,
+        request: Request<api::CertificateRequest>,
```

Comment:
Maybe, we should have a `use api::CertificateRequest` on top and here replace `api::CertificateRequest` by `CertificateRequest`. If only for consistency with the rest of the code.

Reply:
After the rename of `HandleCertificateRequest` to `CertificateRequest`, I can't